### PR TITLE
LF-2617-stars-for-crop-plan-rating-active-without-editing

### PR DESCRIPTION
**Pull request by Duncan-Brain**
Jira ticket: https://lite-farm.atlassian.net/browse/LF-2617?atlOrigin=eyJpIjoiZjcyNTljMDZiODBkNGM0M2EwNmU5YmIyMDZkODA3YTYiLCJwIjoiaiJ9

Copied format of Rating in Task/TaskReadOnly/ to Crop/ManagementDetail/ManagementPlanDetail

### CODE REVIEW

**Comment by anant00729** [comment]
@Duncan-Brain This is not working for me. After abandoning or completing the management plan, I can still edit the ratings.

**Comment by Duncan-Brain** [comment]
@anant00729 That's weird... it seems to be working for me. I am testing at the following path and seeing that the ratings are not editable.

`/crop/<crop_id>/management_plan/<management_plan_id>/details`

Can you try again and maybe include a recording or steps to reproduce if you still see it?

**Comment by anant00729** [comment]
This works for me. You can add the inline CSS properties to the  src/components/Crop/ManagementDetail/styles.module.scss.

**Review by anant00729 — APPROVED** [review]

File: packages/webapp/src/components/Crop/ManagementDetail/ManagementPlanDetail.jsx [hunk 1]
```diff
@@ -132,13 +132,15 @@ export default function PureManagementDetail({
...
-          className={styles.rating}
-          style={{ marginBottom: '34px' }}
-          label={t('MANAGEMENT_PLAN.RATE_THIS_MANAGEMENT_PLAN')}
-          stars={plan.rating}
-          onRate={() => {}}
-        />
+        <div>
+          <Rating
+            className={styles.rating}
+            style={{ marginBottom: '34px', width: '24px', height: '24px' }}
```
[the remark this answers is not in the document]
Comment (Duncan-Brain) [reply packages/webapp/src/components/Crop/ManagementDetail/ManagementPlanDetail.jsx:138]: I just tested this and unfortunately it does not have the desired effect. 

It seems it is actually important to pass this style to a child component holding the stars within rating not to the rating container (containing both the label and the stars).

**Review by wasswarichard — APPROVED** [review]

**Pull request merged**